packages/web-twig/src/Resources/components/Tooltip/README.md

```diff
@@ -38,27 +34,20 @@ Without lexer:
 
 ## Linking with Content
 
-Tooltip is positioned relative to the closest parent element with
-`position: relative` or `position: absolute`. You may either provide the CSS
-yourself or you can use the prepared TooltipWrapper component:
-
 ```html
-<TooltipWrapper>
+<Tooltip>
     <Link href="#" aria-describedby="my-tooltip">
         I have a tooltip
     </Link>
-    <Tooltip id="my-tooltip">
+    <TooltipPopover id="my-tooltip">
         Hello there!
-    </Tooltip>
-</TooltipWrapper>
+    </TooltipPopover>
+</Tooltip>
 ```
 
-Please consult the [CSS implementation of Tooltip][tooltip] to help you pick the
```

Comment:
🤔 I don't know if we should remove this as well. :)

Reply:
Okay, it is back. It's better to have something extra than to be missing something.